src/components/FormContainer.tsx

```diff
@@ -263,11 +296,19 @@ const FormListContainer = <A,>(
 
   return (
     <>
+      <Prompt
+        when={formState === FormState.Dirty}
+        message="Warning: you have unsaved changes. Are you sure you want to leave?"
```

Comment:
I'm not seeing this message show up in the preview environment, is there a section/page I should visit?

Reply:
It looks like it just shows up on pages with the internal item forms, like 1099s, W-2s, spouseanddependent, if the user navigates away using the sidebar while unsaved data is present in the form.

![image](https://user-images.githubusercontent.com/1987109/139637148-ccd53221-30cd-4755-91d2-32173b8dfeaf.png)

Reply:
Gotcha